src/paths.ts

```diff
@@ -95,8 +95,7 @@ class Paths {
             }
           }
         }
-        // We've got nothing...
-        throw new Error('No CMake found on $PATH');
+        return '';
```

Comment:
Instead of empty string, return `null` and tweak the return type of this function. Change callers to check for `null` and handle appropriately.